client/src/components/ElementEditor/Content.js

```diff
@@ -2,13 +2,19 @@ import React, { Component, PropTypes } from 'react';
 
 class Content extends Component {
   render() {
+    const { summary } = this.props;
+
+    // HTML Fragment returned from BaseElement's getEditorPreview()
     return (
-      <div />
+      <div className="element-editor-content" dangerouslySetInnerHTML={{ __html: summary }} />
     );
   }
 }
 
 Content.defaultProps = {};
-Content.propTypes = {};
+Content.propTypes = {
+  summary: PropTypes.string,
+};
 
+export { Content as Component };
```

Comment:
This can be done in a separate pull request since it's not part of the Header. If we're gonna keep it here then I think we need to change BaseElement to return a string for the thumbnail URL and we render it from this end rather than using `dangerouslySetInnerHTML`